pkg/controllers/termination/controller.go

```diff
@@ -64,7 +64,7 @@ func NewController(kubeClient client.Client, cloudProvider cloudprovider.CloudPr
 }
 
 func (c *Controller) Name() string {
-	return "termination"
+	return "node.termination"
```

Comment:
Same here, I think this should just be termination.

Reply:
This one I'm actually convinced _should_ be `node.termination`. Realistically, I could see us putting this in the node directory. I think we should take the convention of prefixing with whatever object that we are acting on.